### PR TITLE
NEXT-33933 - Improved employee invitation docs

### PR DESCRIPTION
**Pull request by MartinKrzykawski**
Updated the employee invitation docs.

Wrote how to invite employee via administration, storefront and store-api and also how to replace the default URL with a custom one.

### CODE REVIEW

File: products/extensions/b2b-components/employee-management/guides/b2b-employee-invitation.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+nav:
+  title: Employee Invitation
+  position: 20
+---
+
+# Employee Invitation
+
+Employees can be created via storefront, administration and store-api to get an invitation email. This has to be confirmed by the employee to set a password.
```
Comment (lukasrump) [inline products/extensions/b2b-components/employee-management/guides/b2b-employee-invitation.md:9]: For me this sentence was a bit akward to read. Maybe something like: 
```suggestion
Employees can be created via storefront, administration and store-api. The employee will get an invitation mail that has to be confirmed to set a password.
```

**Review by lukasrump — APPROVED** [review]
Nice work. I added just one optional comment.

File: products/extensions/b2b-components/employee-management/guides/b2b-employee-invitation.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+## Invitation via Administration
+
+As a merchant, you can create and invite the employees via administration. Select the business partner customer, switch to the `company` tab and add a new employee account in edit mode.
+Because the administration is not related to a sales channel, you can select the sales channel which is used for the invitation. This field is pre-selected and disabled if the business partner customer is bound to a sales channel.
+
+## The URL for the Invitation Acceptance
+
+The employees will get an invitation email, to set a password and to fulfill the registration. This process will also activate the employee for the business partners company.
+The default URL for the acceptance is `/account/business-partner/employee/invite/%%RECOVERHASH%%`, the recovery hash is used as a unique identifier and is only valid for the invitation of one employee.
```
Comment (sushmangupta) [inline products/extensions/b2b-components/employee-management/guides/b2b-employee-invitation.md:25]:
```suggestion
# Employee Invitation

Employees can be created via Storefront, Store-api, and Administration. 

- Storefront - Business partners can invite employees by logging-in to Storefront and navigating to the `employee` page. From there, they can add a new employee.
- Store API - One can utilize the `/store-api/employee/create` endpoint while logged in as a customer to invite employees.
- Administration - Merchants can invite employees by logging in to the administration interface. Selects the business partner customer, navigates to the `company` tab, and adds a new employee account in edit mode.

The invited employee receives an invitation mail that must be confirmed to set a password.

## The URL for the invitation acceptance

Upon invitation, the employee will receive an email requiring confirmation to set a password. This process will also activate the employee for the business partners company.
The default URL for the acceptance is `/account/business-partner/employee/invite/%%RECOVERHASH%%`, the recovery hash is used as a unique identifier and is only valid for the invitation of one employee.
```

**Review by sushmangupta — CHANGES_REQUESTED** [review]
@MartinKrzykawski , please take a look

**Comment by MartinKrzykawski** [comment]
> @MartinKrzykawski , please take a look

Hey, looks good to me, i commited your provided changes. Thanks! :)

**Review by sushmangupta — APPROVED** [review]
Thank you @MartinKrzykawski for the update!

**Pull request merged**